server/src/test/java/io/crate/integrationtests/ForeignDataWrapperITest.java

```diff
@@ -87,6 +87,20 @@ public void test_full_fdw_flow() throws Exception {
         assertThat(response).hasRows(
             "doc| dummy"
         );
+        execute(
+            """
+            SELECT
+                option_name,
```

Comment:
why not selecting all columns?

Reply:
Added foreign_table_schema and foreign_table_name.